h2/src/main/org/h2/command/dml/Query.java

```diff
@@ -459,6 +465,18 @@ static void initOrder(Session session,
         }
     }
 
+    /**
+     * Initialize the order or distinct expression.
```

Comment:
Not necessary distinct, this method is used wider.

Reply:
I see only two usages: initialize order expressions & distinct expressions. What did I miss?

Reply:
Oh, I read it as “order **of** distinct”.

However, it's better to write “order by or distinct” here, because it works with expressions from ORDER BY and DISTINCT clauses.

Reply:
Ok. Fixed.